player.py

```diff
@@ -22,4 +22,7 @@ def get_choice(self,Board):
 
 
 if __name__ == "__main__":
-    Player('a')
\ No newline at end of file
+    p = Player('a')
```

Comment:
Ignore this, I was just testing to see if the self variable name had any effect.